src/components/WindowMiddleContent.js

```diff
@@ -35,12 +35,7 @@ class WindowMiddleContent extends Component {
     const { manifest, window } = this.props;
     return (
       <div className={ns('window-middle-content')}>
-        <WindowSideBar
-          windowId={window.id}
-          manifest={manifest}
-          sideBarOpen={window.sideBarOpen}
-          sideBarPanel={window.sideBarPanel}
-        />
+        <WindowSideBar windowId={window.id} />
```

Comment:
That's how we have to do it. Basically, all subcomponents of <Window/> should only get the `windowId` passed from its parents. The id then can be used to get further things from the store. :+1: